schematics/ng-add/steps/add-bootstrap.ts

```diff
@@ -0,0 +1,82 @@
+import * as path from 'path';
+import {Rule, SchematicContext, Tree} from '@angular-devkit/schematics';
+import {getWorkspace} from '@schematics/angular/utility/config';
+import {getProject} from '@schematics/angular/utility/project';
+import {WorkspaceProject, WorkspaceSchema} from '@schematics/angular/utility/workspace-models';
+
+import {Schema} from '../schema';
+import * as messages from '../messages';
+import {getProjectStyleFile, getProjectTargetOptions} from '../../utils/project';
+
+
+const BOOTSTRAP_CSS_FILEPATH = 'node_modules/bootstrap/dist/css/bootstrap.min.css';
+const SUPPORTED_BOOTSTRAP_STYLE_IMPORTS = {
+  '.sass': `
+/* Importing Bootstrap SCSS file. */
+@import '~bootstrap/scss/bootstrap'
```

Comment:
I assume bootstrap does not have a proper `sass` export here for that ?
Sounds weird to import `scss` into `sass`...

Reply:
Nope it does not. You would prefer to support only `scss` and in the case of `sass`, just install the `bootstrap.min.css` ?

Reply:
No, let's keep it like that. I assume it's working ?

Reply:
It is working, the difference between the two snippets is actually  just the `;` symbol at the end of the import..